amundsen_application/static/js/components/TableDetail/ColumnListItem/ColumnType/index.tsx

```diff
@@ -127,8 +131,8 @@ export class ColumnType extends React.Component<
         >
           <Modal.Header closeButton>
             <Modal.Title>
-              {MODAL_TITLE}
-              <div className="column-name">{columnName}</div>
+              <div className="main-title">{MODAL_TITLE}</div>
```

Comment:
Not sure of how this displays, but we might try to use a lower hierarchy h here?

Reply:
Let me take a look to see if one matches.

Reply:
The `<Modal.Title>` from `react-bootstrap` is already an `h4`, I will use `h5` and it doesn't affect the display since we have no applicable styles on that.